test/integration/targets/copy/tasks/tests.yml

```diff
@@ -14,6 +14,7 @@
     src: foo.txt
     dest: "{{ remote_file }}"
     mode: 0444
+    force: False
```

Comment:
I will recheck this doesn't impact coverage - the yum test suite looks like it may test both the dest not existing and it existing https://github.com/ansible/ansible/blob/1e7f7875c617a12e5b16bcf290d489a6446febdb/test/integration/targets/yum/tasks/yum.yml#L179-L201

```suggestion
```